hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -192,8 +199,23 @@ public SingleTransactionRecordBuilderImpl(@NonNull final Instant consensusNow) {
      */
     public SingleTransactionRecordBuilderImpl(
             @NonNull final Instant consensusNow, final ReversingBehavior reversingBehavior) {
+        this(consensusNow, reversingBehavior, null);
+    }
+
+    /**
+     * Creates new transaction record builder with both explicit reversing behavior and
+     * transaction construction finishing.
+     *
+     * @param consensusNow the consensus timestamp for the transaction
+     * @param reversingBehavior the reversing behavior (see {@link RecordListBuilder}
+     */
+    public SingleTransactionRecordBuilderImpl(
+            @NonNull final Instant consensusNow,
+            @NonNull final ReversingBehavior reversingBehavior,
+            @Nullable final ExternalizedRecordCustomizer customizer) {
```

Comment:
Out of curiosity, why allow it to be nullable?  Almost as easy to provide one, especially given `NOOP_EXTERNALIZED_RECORD_CUSTOMIZER`